changelog/2025.2.0.rst

```diff
@@ -57,13 +57,17 @@ Custom components were deprecated over a year ago and, as promised, support for
 encourage all developers to migrate their custom components to :doc:`/components/external_components`. Please see
 :ref:`a_note_about_custom_components` for a detailed explanation and next steps.
 
-Removal of ``armv7`` Docker Support
------------------------------------
+Removal of ``armv7`` Docker and Home Assistant add-on support
```

Comment:
```suggestion
Removal of ``armv7`` Docker (incl. Home Assistant add-on support for ``armv7``) 
```
Title looks like HA add-on is completely unsupported.

Reply:
Yeah agreed with that comment. The other approach would be just:

```suggestion
Removal of ``armv7`` support
```

Simple, to the point, applies everywhere.

Reply:
If you're going to change the title text, then you need change the underline too.  Otherwise, it can't be trivially committed. :-)